Metoder-Exercise4/Program.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System;
+
+namespace Metoder_Exercise4
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            Console.Write("Skriv ett pris: ");
+
+            int userInput = Convert.ToInt32(Console.ReadLine());
```

Comment:
Tänk på att det kan vara enklare att felsöka sin kod om man delar upp varje funktionsanrop på en egen rad

```suggestion
            string userConsoleInput = Console.ReadLine();
            int userInput = Convert.ToInt32(Console.ReadLine());
```